torch/nn/modules/module.py

```diff
@@ -165,6 +158,10 @@ def register_module_full_backward_hook(
     in :attr:`grad_input` and :attr:`grad_output` will be ``None`` for all non-Tensor
     arguments.
 
+    Note that this hook need to apply a Function to the inputs and outputs of the Module
+    during the forward pass. This means that Tensors passed as argument to the Module and
```

Comment:
Typically words like "Tensors" and "Module" aren't capitalized

Reply:
Ho really? I always capitalize them when they refer to the actual PyTorch objects and not when they refer to general tensor/module.
Should I update it here?

Note that two lines above, there is a capitalized Tensor haha